sdk/keyvault/azure-security-keyvault-jca/README.md

```diff
@@ -1,26 +1,22 @@
-# JCA Provider for Azure Key Vault
+# Azure Key Vault JCA client library for Java
```

Comment:
@mnriem , We can add java code in README.md to `src/samples/java`. So we can:
1. Make sure the java code in README.md can be compiled without error.
2. Make sure the java code in README.md to be latest.

Refs:
1. https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/parents/azure-sdk-parent/pom.xml#L109
2. https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/parents/azure-client-sdk-parent/pom.xml#L1196-L1338

Sample:
1. https://github.com/Azure/azure-sdk-for-java/blame/master/sdk/cosmos/azure-spring-data-cosmos/README.md#L92

Reply:
Fixed